src/modules/telemtry/entities/telemetry.ts

```
@@ -0,0 +1,40 @@
+  @Entity('telemetry')
+  export class Telemetry extends BaseEntity {
+    @PrimaryGeneratedColumn()
+    id: number;
+  
+    @Column({ type: 'bigint', nullable: false })
+    syncts: number;
+  
+    @Column({ type: 'jsonb', nullable: false })
+    params: object;
+  
+    @Column({ type: 'bigint', nullable: false })
```

ESLint

[error] Replace `····` with `··`

(prettier/prettier)